### DISCUSSION TITLE
Possible Fraud Attempt using Github Notifaction

### DISCUSSION THREAD

**Discussion by jerrylususu** [community/community#109422] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

 I am getting an email about "Github Hiring" (as a issue mention) and upon closer inspection this seems like a fraud, and it abuses Github's notification email address to get around spam filter. I want to report that but since I don't have write access on the repo there is just no "Report" button. I searched around on how to report abuse but found nothing useful (I can't even open a support ticket for that!), and this forum is my last attempt.

Here is the suspicious issue: https://github.com/EdwardWason/github-slideshow/issues/1

And to Github: PLEASE add a report spam button for issues/comments/discussions so ANYONE can report it. Current report process is just too supbar compared to other online community.

**Comment by owieth** [comment]
fyi: https://github.com/orgs/community/discussions/109171

**Reply by elGuille** [reply]
This is now a 404.
THIS WAS THE POST WARNING ABOUT THE FRAUD, but it got suspended instead.

What the actual F?

**Comment by crrusucosmin** [comment]
Hi! My GitHub account rusucosmin got disabled after I fell into the trap of this spam message - I need back access to the account, or I would like to get all my private repositories so I can move then somewhere else

**Comment by crrusucosmin** [comment]
Hi,

My GitHub username rusucosmin got disabled because I was a victim of a spam attack. I'm sorry - the notification looked very legitimate, but after I "signed in using GitHub" they started posting comments on Pull Requests on my behalf, with the stolen access token.

Earlier this morning I started receiving a lot of emails about some threads in GitHub that mentioned me. It looked like all of them were some sort of job opportunity threads on Repositories that I contributed to.

Then, My account was disabled. By no means I was involved in this spam, and I believe my account was disabled by mistake. I use GitHub for a lot of business projects, work, personal projects, and education. You can check out my activity.

Please help me unblock my account - I've also attached screenshots of the events...
[Screenshot 2024-02-21 at 9.03.18 AM.png](https://github.zendesk.com/attachments/token/z4Bg56FTU2z6wU4sr1OD1a0hr/?name=Screenshot+2024-02-21+at+9.03.18+AM.png)
[Screenshot 2024-02-21 at 9.03.24 AM.png](https://github.zendesk.com/attachments/token/ZY3DCN0AB3EKNMEw4Pct61F1e/?name=Screenshot+2024-02-21+at+9.03.24+AM.png)
[Screenshot 2024-02-21 at 9.06.28 AM.png](https://github.zendesk.com/attachments/token/ppUBforUe2yIdvFfEXaF2Mv1b/?name=Screenshot+2024-02-21+at+9.06.28+AM.png)
Best,
Cosmin

**Comment by ettaboyle** [comment, marked as answer]
We understand the inconvenience caused by these notifications. Our teams are currently working on addressing these unsolicited phishing notifications. We want to remind our users to continue to use [our abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam) to raise any abusive or suspicious activity.

We would like to bring to our users’ attention to the following:
- Please do not click any links or reply to these notifications. Please [report them](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam).
- [Authorizing an OAuth app](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/authorizing-oauth-apps) can expose your GitHub account and data to a third party.  
- GitHub’s recruitment process would never mention its users via issues/PRs or other public content.
- GitHub recommends you periodically [review your authorized OAuth apps](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/reviewing-your-authorized-oauth-apps). 
This spam activity targets and abuses GitHub’s mention and notification functionality.
- This is a phishing campaign and is not the result of a compromise of GitHub or its systems.

**Comment by lgarron** [comment]
I was also `@`-mentioned in such an issue. It was deleted, which is great.

However, the repo with that contained that issue now lingers at https://github.com/notifications?query= permanently, with an undismissable notification badge:

![clipboard](https://github.com/user-attachments/assets/3f4e6981-75f3-4ec3-802c-ebf001dcc940)

Is there a way for me to fix this?
